app/views/hyrax/embargoes/_list_deactivated_embargoes.html.erb

```diff
@@ -0,0 +1,10 @@
+<% assets_with_deactivated_embargoes.each do |curation_concern| %>
+  <%= link_to curation_concern, edit_embargo_path(curation_concern) %> <%= visibility_badge(curation_concern.visibility) %>
+  <ul>
+    <% curation_concern.embargo_history.each do |entry| %>
+      <% unless entry.empty? %>
```

Comment:
I find such difficulty negligible.  `unless` is particularly useful in guard clauses, e.g.: 
```ruby
next unless obj.condition? && obj.match(/regex/)
```
Where each conditional chunk then reflects an assertion that can be made about the object inside the loop.